NEWS

```diff
@@ -144,6 +144,10 @@ New Functionality
   ``ssh2_gh_gex_init``, ``ssh2_gss_init``, ssh2_rsa_secret``) to detect
   when SSH client and server roles are reversed.
 
+- Analyzer tags added to ``Analyzer::requested_analyzers`` will be enabled
+  at ``zeek_init()`` time. This is primarily useful in combination with
+  ``Analyzer::disable_all`` and external scripts requiring analyzers.
```

Comment:
Might be worth stating here that it's a no-op when `disable_all` remains at its default `F`. I could also see more complex interactions if other code uses `Analyzer::disable_analyzer()` at startup time, but that's likely better covered in the docs — there's still zeek/zeek-docs#151.